### PR TITLE
Part4

### PR DESCRIPTION
**Pull request by HYLLLIC**
Let me know if anything should be adjusted. Thank you!

### CODE REVIEW

File: Wrappers.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "Apartment.h"
+#include "CorporateOffice.h"
+#include "ElectricHeater.h"
+#include "FieldOffice.h"
+#include "PhoneBook.h"
```
Comment (matkatmusic) [inline Wrappers.h]: Don't include what you don't use
remove these include statements, move them to the cpp file, unless they absolutely have to be here
- [x] revise

File: PhoneBook.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    {
+        int pagesToTear = (numberOfPages - pagesTorn) / 2 + 25;
+        if (pagesTorn + pagesToTear > numberOfPages) 
+        {
+            pagesTorn = numberOfPages; 
+        }
+        {
+            pagesTorn += pagesToTear;
+            std::cout << "Tearing out " << pagesToTear << " pages." << std::endl;
+        }
```
Comment (matkatmusic) [inline PhoneBook.cpp]: Why are there extra curly braces here?
- [x] revise

File: CorporateOffice.cpp [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+{
+    securityGuard.name = "THE BOSS, NOW";
+}
+
+float CorporateOffice::Employee::negotiateRaise(float raiseAmount)
+{
+    float newSalary(salary);
+    while (newSalary < salary + raiseAmount)
+    {
+        newSalary+=(raiseAmount*.25f);
```
Comment (matkatmusic) [inline CorporateOffice.cpp]: add space around your math operators to make the code easier to read
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: PhoneBook.cpp (hunk 2 above)
Comment (HYLLLIC) [reply PhoneBook.cpp]: The extra brackets removed. Also reconfigured the function as the "pagesToTear" variable was unnecessary and a bit weird. I'll be sure to reformulate old stuff as I go forward - initial function created with a misunderstanding of "if" syntax.

File: Wrappers.h (hunk 1 above)
Comment (HYLLLIC) [reply Wrappers.h]: Only moving "FieldOffice.h" caused an issue. All other header includes moved to cpp file.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**